docs/source/user_guide.rst

```diff
@@ -30,7 +30,7 @@ For example, we can define a random circuit in Qiskit and optimize it using the
 
    from qiskit.circuit.random import random_clifford_circuit
    import ucc
-   from ucc.benchmarks.utils import count_multi_qubit_gates_qiskit
+   from benchmarks.utils import count_multi_qubit_gates_qiskit
```

Comment:
For me, I had to switch this to `from benchmarks.scripts.common` as the utils submodule didn't exist.

But also curious, do we want benchmarks to be a submodule of ucc or a sibling?

Reply:
Thanks Brad. I think it's more intuitive to have benchmarks as a submodule of ucc. I think the benchmarks module was intentionally made a sibling of ucc during earlier restructuring of the library, but I don't remember the reason.